spec/sidekiq-instrument/server_middleware_spec.rb

```diff
@@ -14,30 +14,66 @@
       end
     end
 
-    it 'increments dequeue counter' do
-      expect {
+    EXPECTED_DOG_OPTIONS = { tags: ['queue:default', 'worker:MyWorker'] }
```

Comment:
instead of defining a constant, we can follow rspec convention of defining a test variable. 
```
let(:expected_dog_options) { ... }
```